app/server/app/public/widgets/communityWidget.html

```diff
@@ -0,0 +1,231 @@
+        margin-left: 6px;
+        padding: 4px 8px;
+        font-weight: bold;
+        text-shadow: 0 0 6px rgba(0, 0, 0, 0.375);
+        color: #fff;
+        background-color: #044b8d;
+      }
+
+      .widget-footer {
+        display: flex;
+        align-items: flex-end;
+        justify-content: space-between;
+        position: absolute;
+        bottom: 0;
+        width: calc(100% - 12px);
+        padding: 6px;
+      }
+
+      .widget-footer img {
+        height: 16px;
+        width: auto;
+      }
+    </style>
+  </head>
+  <body class="blank">
+    <!-- Google Tag Manager -->
+    <noscript>
+      <iframe
+        title="google tag manager"
+        src="//www.googletagmanager.com/ns.html?id=GTM-L8ZB"
+        height="0"
+        width="0"
+        style="display: none; visibility: hidden">
+      </iframe>
+    </noscript>
+    <script>
+      (function (w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : '';
+        j.async = true;
+        j.src = '//www.googletagmanager.com/gtm.js?id=' + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, 'script', 'dataLayer', 'GTM-L8ZB');
+    </script>
+    <!-- End Google Tag Manager -->
+
+    <noscript>
+      <style>#hmw-form { visibility: hidden; }</style>
+    </noscript>
+
+    <div class="widget">
+      <div class="widget-content">
+        <h1>
+          <a href="https://mywaterway.epa.gov/" target="_blank" rel="noopener noreferrer">
+            How’s My Waterway
+          </a>
+        </h1>
+
+        <p>Explore water quality in your community!</p>
+
+        <form id="hmw-form">
+          <label for="hmw-location">Enter a location:</label>
+          <div class="widget-input">
+            <input
+              id="hmw-location"
+              type="text"
+              name="location"
+              value="address, zip or place"
+              onfocus="clearDefault(this)"
+              onblur="setDefault(this)"
+            />
+            <input type="submit" value="Go" />
+          </div>
+        </form>
+      </div>
+
+      <div class="widget-footer">
+        <a href="https://www.epa.gov/waterdata/hows-my-waterway">
+          <img src="https://www.epa.gov/sites/production/files/2015-06/info.png" alt="How's My Waterway Information" />
+        </a>
+        <a href="https://www.epa.gov">
+          <img src="https://www.epa.gov/sites/production/files/2014-03/epa-logo.png" alt="EPA Home Page" />
+        </a>
+      </div>
+    </div>
+
+    <script>
+      var form = document.getElementById('hmw-form');
+
+      if (form.addEventListener) {
+        form.addEventListener('submit', onSubmit, false); //Modern browsers
+      } else if (form.attachEvent) {
+        form.attachEvent('onsubmit', onSubmit); //Old IE
+      }
```

Comment:
@cschwinderg I left this in, but we probably don't need support for old IE anymore, and can just use `addEventListener`. And if we did that, we might as well remove the `onfocus` and `onblur` attributes from the text input and add event listeners via JS alone, like we're doing for the `onSubmit()` method.

Reply:
I just added this to support old IE browsers because the other widgets seem to support old IE browsers. See lines 1 - 6 of the `communityWidget.html` file.